datajoint/__init__.py

```diff
@@ -41,5 +41,6 @@
 from .attribute_adapter import AttributeAdapter
 from . import errors
 from .errors import DataJointError
+from .migrate import migrate_dj011_external_blob_storage_to_dj012
```

Comment:
not so important, not sure if we want to import by default..

Reply:
Right, good point. This can be safely removed.